version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.8.3-rc.{height}",
```

Comment:
Please revert this change. Main should use `2.8.3-rc.{height}` format.

Reply:
yes